src/v2/BaseLoginRouter.js

```diff
@@ -185,6 +186,18 @@ export default Router.extend({
     // }
   },
 
+  handleIdxInteractionCanceled() {
```

Comment:
instead of creating this new handler, it should use existing `startLoginFlow` of `render`